third_party/openthread/platforms/efr32/BUILD.gn

```diff
@@ -49,21 +53,21 @@ source_set("openthread_core_config_efr32") {
 
 source_set("libopenthread-efr32") {
   sources = [
-    "${openthread_efr32_root}/openthread/examples/apps/cli/cli_uart.cpp",
-    "${openthread_efr32_root}/src/src/alarm.c",
-    "${openthread_efr32_root}/src/src/crypto.c",
-    "${openthread_efr32_root}/src/src/entropy.c",
-    "${openthread_efr32_root}/src/src/flash.c",
-    "${openthread_efr32_root}/src/src/misc.c",
-    "${openthread_efr32_root}/src/src/radio.c",
-    "${openthread_efr32_root}/src/src/security_manager.c",
-    "${openthread_efr32_root}/src/src/sleep.c",
-    "${openthread_efr32_root}/src/src/system.c",
+    "${openthread_efr32_root}/alarm.c",
```

Comment:
so when building from ot-efr32 submodule your `openthread_efr32_root` needs to include src/src right?

Reply:
yes that is correct